components/toast/index.css

```diff
@@ -30,68 +72,120 @@ governing permissions and limitations under the License.
   flex-direction: row;
   align-items: stretch;
 
-  border-radius: var(--spectrum-toast-neutral-border-radius);
+  border-radius: var(--mod-toast-corner-radius, var(--spectrum-toast-corner-radius));
+
+  padding-inline-start: var(--mod-toast-spacing-start-edge-to-text-and-icon, var(--spectrum-toast-spacing-start-edge-to-text-and-icon));
 
-  padding-block: var(--spectrum-toast-neutral-padding-y);
-  padding-inline-end: var(--spectrum-toast-neutral-padding-right);
-  padding-inline-start: var(--spectrum-toast-neutral-padding-left);
+  font-size: var(--mod-toast-font-size, var(--spectrum-toast-font-size));
+  font-weight: var(--spectrum-toast-font-weight);
 
-  font-size: var(--spectrum-toast-neutral-text-size);
-  font-weight: var(--spectrum-toast-neutral-text-font-weight);
   -webkit-font-smoothing: antialiased;
+
+  background-color: var(--mod-toast-background-color-default, var(--spectrum-toast-background-color-default));
+  color: var(--mod-toast-background-color-default, var(--spectrum-toast-background-color-default));
+
+  &--negative,
+    /* @deprecated */.spectrum-Toast--error {
```

Comment:
remove if deprecated?

Reply:
This is a great question that we should maybe surface to design and/or Jess. I'm inclined to think that we should only show current guidance and get rid of the "use X variant instead" examples in the docs.

Reply:
Asked [here](https://adobedesign.slack.com/archives/G019JTYMT6H/p1659387331617259)

Reply:
I removed the deprecated variants.